packages/salesforcedx-vscode-lwc-next/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "salesforcedx-vscode-lwc-next",
```

Comment:
Here's the new name. Once we go GA, this one will replace the salesforcedx-vscode-lwc.